content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Fission Function Orchestration with Argo Workflows"
+date = "2022-09-21T10:15:00+05:30"
+author = "Neha Gupta"
+description = "Multiple Fission function orchestration using Argo Workflows"
+categories = ["Tutorials"]
+type = "blog"
+images = ["images/featured/fission-zapier-featured.png"]
++++
+
+Fission Functions allow user to perform one logical task. To group multiple task together such as one function is dependent on other we can use Argo Workflows.
+Argo Workflows is an open source container-native workflow engine with a feature to create DAGs i.e. running task sequentially, in parallel and with dependencies. We will try to develop a simple Insurance Eligibility program which will take different input and calculate insurance installment on basis of the inputs.
+
+### Understanding the Insurance Calculator
+
+The small application/Workflow will take basic information from the user and check the eligibility for the Insurance. Below are the input and different tasks which are performed to calculating the above:
+
+* Take first name of the user
+* Take last name of the user
+* Create full name using the above two.
+* Input age and monthly salary.
+* On basis of predefined criteria i.e. ,
+  * if age is below a certain age and yearly salary is above some numbers, user lies in a low risk criteria
+  * otherwise he/she falls in high risk criteria. And the installments varies accordingly.
+* We print the above calculated output to the user.
+
+{{< figure src="/images/featured/insurance-workflow-argo.png" alt="Developing Insurance calculator DAG using Argo Workflows" height="400" width="600">}}
+
+## Pre Requisites
+
+### Fission
+
+You should have Fission running on your system. You can refer to our [Fission Installation](/docs/installation) guide for more information.
+
+### Argo Workflows
+
+For installing the latest version Argo Workflows, you can refer to [Argo Workflows Installation](https://argoproj.github.io/argo-workflows/quick-start/)  
+
+We are ready to build our Insurance Installment calculator now.
+
+### Creating Fission functions and triggers
+
+We have to create all the functions which would be helping us with the calculation. You can refer to [Fission Function](https://github.com/neha-Gupta1/argo-workflow/tree/main/insurance/fission-functions) folder for all the functions required for this application. Clone the above folder and run below command -
+
+`sudo chmod 755 insurance/fission-functions/spec.sh`
+
+Verifying the Fission function and Trigger creation,
```

Comment:
Let's replace this line with something like this "Verify that all the required functions and triggers are created:"